README.md

```diff
@@ -50,7 +50,7 @@
 
```

Comment:
- [OPTIONAL ] **Please I'M Not able to inline comment in all your README**  but here is the issue : You've done a great job so far on this project, but you still need to improve on something for it to be approved. I noticed that your  `README` file the  M.I.T License is not really working since you provide the wrong link , please include it because using the MIT License in our projects as students at Microverse can provide us  with a number of benefits, including collaboration opportunities, improved skills, portfolio building, career opportunities, and networking., kindly make sure that your M.I.T license is working properly :wink: 
![Screenshot from 2023-02-16 18-23-15](https://user-images.githubusercontent.com/88894525/219427148-0a763978-871f-4a7a-9bed-3f3b71496e3d.png)